tapiro/core/src/test/scala/io/buildo/tapiro/TapiroSuite.scala

```diff
@@ -0,0 +1,135 @@
+package io.buildo.tapiro
+
+import java.nio.file.Files
+
+class TapiroSuite extends munit.FunSuite {
+
+  check(
```

Comment:
This is a test, the idea is that you pass in the controller code and the test runs tapiro and checks the expected code against the generated one.